pkg/diff/diff.go

```diff
@@ -450,21 +462,26 @@ func (t *TableDiff) checkChunkDataEqual(ctx context.Context, filterByRand bool,
 
 func (t *TableDiff) compareChecksum(ctx context.Context, chunk *ChunkRange) (bool, error) {
```

Comment:
we can get the source and target checksum concurrent?

Reply:
do it in #370